edk2toolext/environment/uefi_build.py

```diff
@@ -125,6 +134,14 @@ def RetrievePlatformCommandLineOptions(self, args):
             self.SkipPostBuild = True
             self.FlashImage = False
 
+    def DeclareCriticalPlatformEnv(self) -> list[PlatformEnv]:
```

Comment:
This function should be moved down near https://github.com/tianocore/edk2-pytool-extensions/blob/b83d8c5fbf7398a5dd1a1cacdf00b1afc326375b/edk2toolext/environment/uefi_build.py#L516

And Probably start with "Platform" and have classmethod decorator (although i don't know if this does anything beyond the default anymore).

Reply:
Renamed to `SetPlatformDefaultEnv()` as it seems like an intuitive and accurate name. The variables returned from `SetPlatformDefaultEnv()` are now also the last variables to be set in `SetEnv()`.